cmd/downloader/downloader/downloader.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ledgerwatch/erigon/common"
 	"github.com/ledgerwatch/log/v3"
 	mdbx2 "github.com/torquem-ch/mdbx-go/mdbx"
+	"go.uber.org/atomic"
```

Comment:
Are you sure you don't want `sync/atomic` instead?

Reply:
Yes. It’s package with typed wrappers for atomic.